js/web/lib/onnxjs/attribute.ts

```diff
@@ -12,6 +12,7 @@ import ortFbs = onnxruntime.experimental.fbs;
 
 export declare namespace Attribute {
   export interface DataTypeMap {
+    boolean: boolean;
```

Comment:
I think this change is incorrect. Firstly this file is not related to the PR ( it's in onnxjs/ folder and should have nothing to do with webgpu ) , and secondly in onnx spec we don't have boolean attribute.